Gemfile.lock

```diff
@@ -14,4 +14,4 @@ DEPENDENCIES
   rake
 
 BUNDLED WITH
-   1.11.2
+   1.17.3
```

Comment:
This will show warning on exisiting users since the bundle version used and now is changed.